docs/installing-northstar/troubleshooting.md

```diff
@@ -99,6 +99,8 @@ If you get this error you can fix it by [installing vcredist 2015-2022 (`vcredis
 
 Go to Mods at the bottom of your screen on the main menu, then click Authentication Agreement and accept
 
+If this doesn't work, check in your [Titanfall2 directory](troubleshooting.md#game-location) for a file called `ns_startup_args`, open it up, and if you see something like `+ns_has_agreed_to_token 2`, change this `2` to a `0`. Don't touch anything else. This way, you can choose whether or not you want to authenticate with Northstar.
```

Comment:
This usually shouldn't happen. As in `+ns_has_agreed_to_token` should never be in `ns_startup_args` during normal usage. Rather this is a product of the auto-matchmaking mod that tends to break a lot of Northstar related things.

And in that case the best approach would be to remove that string manually, then launch Northstar, go to mods, and trigger authentication agreement again.

Reply:
it was an issue for a few weeks again but died out again and with the pr to remove functionality anyways, this is a skill issue pr